pkg/test/framework/components/istio/eastwest.go

```diff
@@ -48,6 +48,7 @@ func (i *operatorComponent) deployEastWestGateway(cluster resource.Cluster, revi
 		return err
 	}
 
+	fmt.Println("linsun: revision is " + revision)
```

Comment:
Assume this is forgotten

Reply:
Yes, still troubleshooting test failures.  Got 1 test pass, but the security test still failed.